Gemfile

```diff
@@ -34,10 +34,10 @@ group :development, :test do
   gem 'fakeweb', '~> 1.3.0'
   gem 'capybara',                   '<= 2.0.2'
   gem 'capybara-webkit', '~> 1.1.1'
+  gem 'jazz_hands'
```

Comment:
Ruby already has a great debugger, why use `jazz_hands`?

You know this gem has increased the time to load the rails?